src/components/cylc/gscan/GScan.vue

```diff
@@ -79,6 +79,25 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
             </v-list>
           </v-card>
         </v-menu>
+        <!-- scan -->
+        <div>
+          <!-- button to scan the filesystem -->
+          <v-tooltip text="Scan filesystem (refresh workflows)" >
+            <template v-slot:activator="{ props }"  >
+              <v-btn
+              icon
+              class="flex-grow-0 flex-column pt-1"
+              id="c-gscan-scan-tooltip-btn"
+              variant="text"
+              size="medium"
+              data-cy="gscan-scan-btn"
+              v-bind="{ props}"
+              @click="scanFilesystem()">
+                <v-icon size="x-large">{{ $options.icons.mdiFolderRefresh }}</v-icon>
+              </v-btn>
+            </template>
+          </v-tooltip>
+        </div>
```

Comment:
Can simplify this a bit as we have configured the default [activator](https://vuetifyjs.com/en/components/overlays/#activator-prop) for `VTooltips` as the parent component.

Also made the button the same size as the filter button next to it

```suggestion
        <v-btn
          icon
          id="c-gscan-scan-tooltip-btn"
          variant="text"
          size="small"
          data-cy="gscan-scan-btn"
          @click="scanFilesystem()"
        >
          <v-icon size="x-large">{{ $options.icons.mdiFolderRefresh }}</v-icon>
          <v-tooltip text="Refresh workflows list" />
        </v-btn>
```